themes/socialblue/assets/css/brand.css

```diff
@@ -1,268 +0,0 @@
-[type="radio"]:checked + label:before {
```

Comment:
Why are we removing brand.css? Does this not break the theme customisation (for SaaS)?

Reply:
I checked this and look like all styles, what was in brand css are present in theme styles. This only override styles from theme and broke styles for button hover, focus state. I think there was made before and now it not needed. If you know some specific page, where it can be broken - let me know.

Reply:
@Crazylord It indeed duplicated some styles from the theme because the color module would parse this file and replace those colors with the configuration of the theme.

- Check out the current 8.x-1.x branch of Open Social
- Enable socialblue and set as default
- In the theme settings change the theme colors
- Observe that this properly works

- Check out this feature branch of Open Social
- Enable socialblue and set as default
- In the theme settings change the theme colors
- I believe that those changes are now broken (if not then maybe @left23 knows what changed?)

Reply:
Maybe then we need only add this file and doesn't include for theme?
So, we will be have some separate file with colors should be use only for theme colors module.
I will check this.